charts/victoria-metrics-agent/templates/statefulset.yaml

```diff
@@ -0,0 +1,145 @@
+{{- if eq .Values.kind "StatefulSet" }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "chart.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "chart.labels" . | nindent 4 }}
+  {{- with .Values.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  serviceName: {{ include "chart.fullname" . }}
+  updateStrategy:
```

Comment:
please don't specify the strategy if it's not defined

Reply:
fixed, please, check it again